noetic/distribution.yaml

```diff
@@ -4004,6 +4004,21 @@ repositories:
       url: https://github.com/nerian-vision/nerian_stereo.git
       version: master
     status: developed
+  nmea_comms:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/nmea_comms.git
```

Comment:
It would be great if you can add a top level license file to the repo, but considering this is already part of previous distributions I will proceed to merge